workflows/illumina.nf

```diff
@@ -79,7 +79,7 @@ workflow ILLUMINA {
         ch_spades_assemblies = METASPADES.out.assembly
             .map { meta, assembly ->
                 def meta_new = meta.clone()
-                meta_new.assembler  = "SPAdes"
+                meta_new.assembler  = "metaSPAdes"
```

Comment:
was this missed in a previous PR? or does this just change the name? and is it parameterized so you could pick a different assembler easily in the future?

Reply:
it just changes the name, I caught it while going through these mapping things. and it isn't parameterized to pick a different assembler because I don't have support for other assemblers at the moment...